plugins/main/public/components/security/policies/policies-table.tsx

```diff
@@ -36,10 +42,13 @@ export const PoliciesTable = ({
             },
           },
         );
-        const data = (response.data || {}).data;
+        const data = response?.data?.data;
```

Comment:
Be mindful that data can be `undefined`.

Reply:
done